database/seeds/RustServiceTableSeeder.php

```diff
@@ -0,0 +1,262 @@
+     * @var array
+     */
+    protected $option = [];
+
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $this->addCoreService();
+        $this->addCoreOptions();
+        $this->addVariables();
+    }
+
+    private function addCoreService()
+    {
+        $this->service = Service::updateOrCreate([
+            'author' => config('pterodactyl.service.core'),
+            'folder' => 'rust',
+        ], [
+            'name' => 'Rust',
+            'description' => 'The only aim in Rust is to survive. To do this you will need to overcome struggles such as hunger, thirst and cold. Build a fire. Build a shelter. Kill animals for meat. Protect yourself from other players, and kill them for meat. Create alliances with other players and form a town. Do whatever it takes to survive.',
+            'startup' => './RustDedicated -batchmode +server.port {{SERVER_PORT}} +server.identity "rust" +rcon.port {{RCON_PORT}} +rcon.web true +server.hostname \"{{HOSTNAME}}\" +server.level \"{{LEVEL}}\" +server.description \"{{DESCRIPTION}}\" +server.url \"{{URL}}\" +server.headerimage \"{{SERVER_IMG}}\" +server.worldsize \"{{WORLD_SIZE}}\" +server.seed \"{{SEED}}\" +server.maxplayers {{MAX_PLAYERS}} +rcon.password \"{{RCON_PASS}}\" {{ADDITIONAL_ARGS}}',
+            'index_file' => Service::defaultIndexFile(),
+        ]);
+    }
+
+    private function addCoreOptions()
+    {
+        $script = <<<'EOF'
+apt update
+apt -y --no-install-recommends install curl lib32gcc1 ca-certificates
+
+cd /tmp
+curl -sSL -o steamcmd.tar.gz http://media.steampowered.com/installer/steamcmd_linux.tar.gz
+
+mkdir -p /mnt/server/steam
+tar -xzvf steamcmd.tar.gz -C /mnt/server/steam
+cd /mnt/server/steam
+
+chown -R root:root /mnt
+
+export HOME=/mnt/server
+./steamcmd.sh +login anonymous +force_install_dir /mnt/server +app_update 258550 +quit
+
+mkdir -p /mnt/server/.steam/sdk32
+cp -v linux32/steamclient.so ../.steam/sdk32/steamclient.so
+EOF;
+
+        $this->option['rustvanilla'] = ServiceOption::updateOrCreate([
+            'service_id' => $this->service->id,
+            'tag' => 'rustvanilla',
+        ], [
+            'name' => 'Vanilla',
+            'description' => 'Vanilla Rust server.',
+            'docker_image' => 'tenten8401/pterodactyl-rust',
+            'config_startup' => '{"done": "Server startup complete", "userInteraction": []}',
+            'config_files' => '{}',
+            'config_logs' => '{"custom": false, "location": "latest.log"}',
+            'config_stop' => 'quit',
+            'config_from' => null,
+            'startup' => null,
+            'script_install' => $script,
+            'script_entry' => 'bash',
+            'script_container' => 'ubuntu:latest',
+        ]);
+    }
+
+    private function addVariables()
+    {
+        $this->addVanillaVariables();
+    }
+
+    private function addVanillaVariables()
+    {
+        ServiceVariable::updateOrCreate([
```

Comment:
Sorry, this comment was specifically directed at this `LD_LIBRARY_PATH` variable.